web-client/src/components/WorkForReviewComponent.jsx

```diff
@@ -0,0 +1,394 @@
+            width: '100%',
+            float: 'right',
+            fontSize: '12px',
+            color: 'grey',
+        },
+        authors:
+        {
+            width: '45%',
+            float: 'right',
+            paddingLeft: '5%',
+        },
+        other:
+        {
+            width: '100%',
+            float: 'left',
+            textAlign: 'left',
+            color: 'grey',
+        },
+        otherName:
+        {
+            width: '100%',
+            float: 'left',
+            textAlign: 'left',
+            marginTop: '6%',
+            fontWeight: 'bold',
+        },
+        text:
+        {
+            width: '85%',
+            float: 'left',
+            textAlign: 'left',
+            paddingLeft: '5%',
+        },
+        btn:
+        {
+            textTransform: 'none',
+        },
+        btn1:
+        {
+            textTransform: 'none',
+            marginLeft: '5%',
+            float: 'left',
+            marginBottom: '2%',
+        },
+        btn2:
+        {
+            textTransform: 'none',
+            marginLeft: '5%',
+            float: 'left',
+            marginBottom: '2%',
+            marginRight: '2.5%',
+        },
+        mainDialog:
+        {
+            maxWidth: '100%',
+        },
+        dialogTitle:
+        {
+            fontWeight: 'bold',
+            textAlign: 'center',
+        },
+        range:
+        {
+            width: '99%',
+            textAlign: 'center',
+            float: 'center',
+            margin: 'auto',
+        },
+        grade:
+        {
+            float: 'left',
+        },
+        stars:
+        {
+            float: 'left',
+            marginLeft: '10%',
+        },
+        label:
+        {
+            float: 'left',
+            marginLeft: '10%',
+        },
+        info:
+        {
+            width: '100%',
+            textAlign: 'center',
+            fontSize: '14px',
+            margin: 'auto',
+            paddingTop: '1%',
+        },
+        textField:
+        {
+            width: "100%",
+        },
+    });
+
+    const labels = {
+        1: 'rejected',
+        2: 'correct',
+        3: 'accepted'
+    };
+
+    const style = styles();
+
+    const [open, setOpen] = React.useState(false);
+    const [value, setValue] = React.useState(1);
+    const [hover, setHover] = React.useState(1);
+
+    const [_file, SetFile] = React.useState(null);
+    const passFile = (f) => {
+        SetFile(f);
+    };
+
+    const [values, setValues] = React.useState({
+        commit: "",
+    });
+
+    const handleClickOpen = () => {
+        setOpen(true);
+        counts.comment = 0;
+        SetFile(null);
+    };
+
+    const handleClose = () => {
+        setOpen(false);
+    };
+
+    const handleChange = (prop) => (event) => {
+        setValues({ ...values, [prop]: event.target.value });
+        setCounts({ ...counts, [prop]: event.target.value.length });
+    };
+
+    //Form
+    const maxCommentSize = 255;
```

Comment:
Hmn?

Reply:
?

Reply:
information for me when the elements related to form validation begin